src/main/java/codesquad/codestagram/domain/User.java

```diff
@@ -55,4 +55,8 @@ public void updateUser(String name, String email) {
         this.email = email;
     }
 
+    public boolean checkEqualTo(String password) {
```

Comment:
```suggestion
    public boolean matchPassword(String password) {
```
요 정도 이름이 더 깔끔할 것 같습니다.